config/default.yml

```diff
@@ -153,6 +153,12 @@ Rails/Date:
     - strict
     - flexible
 
+Rails/DefaultScope:
+  Description: 'Looks for using of `default_scope`.'
```

Comment:
Grammar: "Looks for _use_ of `default_scope`.

(but for consistency with other cops, I think "Avoid use of `default_scope`" would be better).

Reply:
>  I think "Avoid use of `default_scope`" would be better

Yeah, I agree with this opinion.